kong/plugins/grpc-gateway/deco.lua

```diff
@@ -1,6 +1,7 @@
 -- Copyright (c) Kong Inc. 2020
 
-require"lua_pack"
+package.loaded.lua_pack = nil   -- BUG: why?
```

Comment:
yeah i faced the same issue as well, i think it's related to some plugin requires this first. but can't find a reasonable explaination